regen/embed.pl

```diff
@@ -110,7 +110,7 @@ sub generate_proto_h {
         my $has_context = ( $flags !~ /T/ );
         my $never_returns = ( $flags =~ /r/ );
         my $binarycompat = ( $flags =~ /b/ );
-        my $commented_out = ( $flags =~ /m/ );
+        my $has_mflag = ( $flags =~ /m/ );
```

Comment:
I think `$is_macro` would be a better name, to better match the other "purpose of flag" names.

Reply:
The problem is that other flags imply that it is a macro  But now that I think about it, it is probably best if those flags cause `m` to be added behind the scenes when it isn't actually present